addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -116,6 +116,9 @@
 
```

Comment:
Detail: Commit message says `related commit: https://github.com/odoo/odoo/pull/152263`, it's a related PR :eyes: